keyboards/preonic/keymaps/kjwon15/keymap.c

```diff
@@ -0,0 +1,375 @@
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      | WHL_U|Mclick|      |      |      | LCTL | MS_U | WHL_U|      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      | WHL_D|Rclick|Lclick|      |      | MS_L | MS_D | MS_R |      | ACL0 |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      | WHL_D| WHL_L| WHL_R|      | ACL1 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      | ACL2 |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_MOUSE] = LAYOUT_preonic_grid( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+  _______, _______, KC_WH_U, KC_BTN3, _______, _______, _______, KC_LCTL, KC_MS_U, KC_WH_U, _______, KC_DEL,  \
+  _______, _______, KC_WH_D, KC_BTN2, KC_BTN1, _______, _______, KC_MS_L, KC_MS_D, KC_MS_R, _______, KC_ACL0, \
+  _______, _______, _______, _______, _______, _______, _______, KC_WH_D, KC_WH_L, KC_WH_R, _______, KC_ACL1, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_ACL2  \
+  )
+
+
+};
+
+static bool cntl_interrupted = 0;
+static uint16_t cntl_timer = 0;
+
+static bool mouse_interrupted = 0;
+static uint16_t mouse_timer = 0;
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+        case QWERTY:
+          if (record->event.pressed) {
+            set_single_persistent_default_layer(_QWERTY);
+          }
+          return false;
+          break;
+        case PURE:
+          if (record->event.pressed) {
+            set_single_persistent_default_layer(_PURE);
+          }
+          return false;
+          break;
+        case BLANK:
+          if (record->event.pressed) {
+            set_single_persistent_default_layer(_BLANK);
+          }
+          return false;
+          break;
+        case LOWER:
+          if (record->event.pressed) {
+            layer_on(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+          } else {
+            layer_off(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+          }
+          return false;
+          break;
+        case RAISE:
+          if (record->event.pressed) {
+            layer_on(_RAISE);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+          } else {
+            layer_off(_RAISE);
+            update_tri_layer(_LOWER, _RAISE, _ADJUST);
+          }
+          return false;
+          break;
+        case BACKLIT:
```

Comment:
You're not using this keycode, so this whole `case` block can be safely removed.